### ISSUE TITLE
iPhone XS Max status bar data missing

### ISSUE THREAD

**Issue by Aransh** [midnightchip/Asteroid#72]
It comes and goes randomly, not sure what's it related to but I'm getting this a lot, looks like it happens in certain apps (Facebook Messenger for example) but it might be because of the location services icon... Not sure 
![527956AF-C463-4C46-B080-DE3A2EE670EC](https://user-images.githubusercontent.com/11724502/57188836-f52aed80-6f0d-11e9-8d8c-1750d970fa42.jpeg)

**Comment by the-casle** [comment]
The carrier text becomes ... ?

**Comment by Aransh** [comment]
No, the carrier text shouldn't be there on on iPhone Xs, it's where the data (3G/4G/LTE) should be
